erpc_c/infra/erpc_transport_arbitrator.cpp

```diff
@@ -48,6 +48,11 @@ void TransportArbitrator::setCrc16(Crc16 *crcImpl)
     m_sharedTransport->setCrc16(crcImpl);
 }
 
+bool TransportArbitrator::hasMessage(void)
+{
+    return m_sharedTransport->hasMessage();
```

Comment:
it would better to put assert before the m_sharedTransport is used, similarly it is done in send and receive methods, like this:
assert(m_sharedTransport && "shared transport is not set");

Reply:
Sorry for the long delay!  I pushed another commit below which adds the missing assert.  Thanks for your feedback.